cmake/modules/FindCUDA.cmake

```diff
@@ -852,6 +852,13 @@ find_package_handle_standard_args(CUDA
     CUDA_VERSION
   )
 
+############################
+# Setup clang cuda
+if (CMAKE_CXX_COMPILER_ID MATCHES Clang 
+    AND RAJA_ENABLE_CLANG_CUDA)
+  add_definitions("-x cuda --cuda-gpu-arch=sm_35 --cuda-path=${CUDA_TOOLKIT_ROOT_DIR}")
```

Comment:
I believe we have verified that everything works with sm_37 or higher.  Should we set this higher?